packages/react-core/src/components/DataList/DataListActionBreakpoints.ts

```diff
@@ -0,0 +1,18 @@
+export enum DataListActionModifiers {
+  hidden = 'hidden',
+  visible = 'visible'
+}
+
+export enum DataListActionBreakpoints {
+  md = 'Md',
+  lg = 'Lg',
+  xl = 'Xl',
+  '2xl' = '2Xl'
+}
+
+export interface DataListActionBreakpointMod {
+  /** The attribute to modify  */
+  modifier: DataListActionModifiers;
```

Comment:
This needs to be`modifier: 'hidden' | 'visible'`  so that values are listed in Documentation.  sam for the line below.

Reply:
No problem. :+1: